extensions/variants.go

```diff
@@ -513,6 +546,14 @@ func (s *WindowFunctionVariant) MatchAt(typ types.Type, pos int) (bool, error) {
 	return matchArgumentAt(typ, pos, s.Nullability(), s.impl.Args, s.impl.Variadic)
 }
 
+func (s *WindowFunctionVariant) MinArgumentCount() int {
+	return minArgumentCount(s.impl.Args, s.impl.Variadic)
```

Comment:
add a test?

Reply:
added tests for this, thanks for pointing this. I missed it